contracts/bridges/RelayManagerETH.sol

```diff
@@ -17,43 +20,64 @@ contract RelayManagerETH is AccessControl, ReentrancyGuard {
   // IDO token address
   IERC20 public ido;
 
-  uint256 public baseGas;
+  // Bridge wallet for collecting fees
+  address public bridgeWallet;
+  // Signer check length threshold
+  uint8 public threshold = 1;
+  // Signer count
+  uint8 public signerLength;
 
+  // Fixed Admin Fee in IDO
   uint256 public adminFee; // bps
   uint256 public adminFeeAccumulated;
-  uint256 public gasFeeAccumulated;
   uint256 public minTransferAmount;
 
+    // address => signer status
+  mapping(address => bool) private _signers;
+
   // Transfer nonce
   mapping(address => uint256) public nonces;
-  // Transfer hash processed status
-  mapping(bytes32 => bool) public processedHashes;
-  // ERC20Permit
-  struct PermitRequest {
-    uint256 nonce;
-    uint256 deadline;
-    uint8 v;
-    bytes32 r;
-    bytes32 s;
-  }
+
+  // transfer from address => nonce => processed status
+  mapping(address => mapping(uint256 => bool)) public processedNonces;
+ 
+  
   // Events
   event Deposited(address indexed from, address indexed receiver, uint256 toChainId, uint256 amount, uint256 nonce);
-  event Sent(address indexed receiver, uint256 indexed amount, uint256 indexed transferredAmount, bytes32 depositHash);
+  event Sent(address indexed receiver, uint256 indexed amount, uint256 indexed transferredAmount, uint256 nonce);
   event AdminFeeChanged(uint256 indexed AdminFee);
   event EthReceived(address indexed sender, uint256 amount);
-  event AdminFeeWithdraw(address indexed receiver, uint256 amount);
-  event GasFeeWithdraw(address indexed receiver, uint256 amount);
+  event BridgeWalletChanged(address indexed bridgeWallet);
+  event ThresholdChanged(uint8 threshold);
+  event SignerAdded(address indexed signer);
+  event SignerRemoved(address indexed signer);
 
   constructor(
```

Comment:
This constructor change caused compilation errors ...
```
TypeError: Wrong argument count for modifier invocation: 2 arguments given but expected 5.

 --> contracts/bridges/RelayManagerETHBatch.sol:9:47:
  |
9 |   constructor(IERC20 _ido, uint256 _adminFee) RelayManagerETH(_ido, _adminFee) { }
```

https://github.com/idexo/ido-contracts/runs/4280572905?check_suite_focus=true#step:7:13